File: Quacks.java
public class Quacks {
    private double QPS; //quacks per second
    private double QPC; //quacks per click
    private double multiplier;

    public Quacks() {
        QPS = 0;
        QPC = 1;
        multiplier = 1;
    }

    /**
     * constrcutor for quack
     * @param QPS quacks per second
     * @param QPC quacks per click
     * @param multiplier the QPS = multiplier * QPS // update later once we actually figure it out
     */
    public Quacks(double QPS, double QPC, double multiplier) {
        this.QPS = QPS;
        this.QPC = QPC;
        this.multiplier = multiplier;
    }

    //getters and setters
    public double getQPS() {
        return QPS;
    }

    public void setQPS(double qPS) {
        QPS = qPS;
    }

    public double getQPC() {
        return QPC;
    }

    public void setQPC(double qPC) {
        QPC = qPC;
    }

    public double getMultiplier() {
        return multiplier;
    }

    public void setMultiplier(double multiplier) {
        this.multiplier = multiplier;
    }
    
}

File: Quacks.js
class Quacks {
  // Constructor without parameters
  constructor(QPS = 0, QPC = 1, multiplier = null) {
    this.QPS = QPS; // quacks per second
    this.QPC = QPC; // quacks per click
    this.multiplier = multiplier; 
    this.quacks = 0;
  }

  // Getters and setters
  getQPS() {      
    return this.QPS;
  }
  incrementQPS() {
    this.QPS++;
  }
  getQPC() {
    return this.QPC;
  }
  setQPC(qpc) {
    this.QPC = qpc;
  }
  getMultiplier() {
    return this.multiplier;
  }
  setMultiplier(multiplier) {
    this.multiplier = multiplier;
  }
  getQuacks() {
    return this.quacks;
  }
  addQuacks(quacks) {
    this.quacks += quacks;
  }
  removeQuacks(quacks) {
    this.quacks -= quacks;
  }
  incrementQuacks() {    
    this.quacks += this.QPC;
  }
}